src/python/py/models/builder.py

```diff
@@ -501,7 +502,7 @@ def to_int4(self, model):
             block_size=self.quant_attrs["int4"]["block_size"],
             is_symmetric=self.quant_attrs["int4"]["is_symmetric"],
             accuracy_level=self.quant_attrs["int4"]["accuracy_level"],
-            nodes_to_exclude=[],
+            nodes_to_exclude=self.quant_attrs["int4"]["exclude_nodes"],
```

Comment:
To match how the other attributes use the same names as the constructor's argument names

```suggestion
            nodes_to_exclude=self.quant_attrs["int4"]["nodes_to_exclude"],
```